tasks/patternpack.js

```diff
@@ -134,6 +139,24 @@ module.exports = function (grunt) {
     }
   }
 
+  function ensureFilesExist(options) {
+    var path = require("path");
+
+    // Create core css file if it does not exist
+    var coreCssPath = options.assets + "/" + options.css.preprocessor + "/" + options.css.fileName + "." + options.css.preprocessor;
```

Comment:
I should clarify - if we allowed `sass` or `scss` as appropriate values in `options.css.preprocessor`, it would only make things more flexible for people who choose to use that syntax (which is generally not recommended these days, even by the Sass team).

The easier option would be to convert the `sass` option to `scss`, which might be sufficient for 1.0.2.